### PR TITLE
Updated workflows tutorial & spurious correlation feature summary to the report section of datalab

### PR DESCRIPTION
**Pull request by allincowell**
## Summary

> Added Spurious Correlations of image-specific features like `dark_score`, `blurry_score`, etc. to `lab.report()` and highlighting the minimum and maximum individual instances in the dataset for each correlated feature.
> The workflows tutorial no longer contains the data generation step to demonstrate spurious correlations for image-specific features. Instead the dataset is directly downloaded from https://s.cleanlab.ai/CIFAR-10-subset.zip.

## Example Usage
```python

from cleanlab import Datalab

lab = Datalab(data=dataset, label_name="label", image_key="image")
lab.find_issues()
lab.report()
```

## Links to Relevant Issues or Conversations

> Earlier related PR: https://github.com/cleanlab/cleanlab/pull/1140

## Outputs on CIFAR-10 subset having original and darkened images

The last two cells of the `workflows.ipynb` tutorial display the output of the CIFAR-10 subset. This includes both the original images and the modified images where one of the classes has been darkened. Here is the [link](https://gist.github.com/allincowell/c334e6a1393e5fe5934a2fd16d284738)

This internal sandbox notebook contains the spurious correlation data-frame on various real-world datasets like `CIFAR-10`, `CIFAR-100`, `Caltech-256`, `Fashion MNIST`, `SVNH`, `Beans`, `Alzheimer MRI` and `100 Sports`. Here is the link: https://github.com/cleanlab/sandbox/blob/main/SpuriousCorrelationsRealWorldDatasets/testing_large_datasets.ipynb. These datasets do not have spurious correlations with class labels which is what's expected.

### CODE REVIEW

**Comment by jwmueller** [comment]
Could you update the PR description to include: 
- what the output of the code snippet looks like for a dataset with known spurious correlation
- what the output of the code snippet looks like for a dataset known to have no spurious correlation
- what tests/benchmarks you ran to ensure this code works across different types of datasets

Thanks!

**Comment by elisno** [comment]
I've made an update to the workflow tutorial.

The code looks fine, but needs more testing.
@allincowell, can you ensure that the reporting code gets better coverage?
1. One test to ensure that for a detected spurious correlation, it should be shown in the report.
2. Another test to ensure that when no spurious correlation is detected, the report shouldn't show anything related to spurious correlations.

File: cleanlab/datalab/internal/adapter/imagelab.py [hunk 1]
```diff
@@ -161,14 +164,63 @@ def __init__(
...
+    def _get_correlated_properties(self) -> List:
+        if self.correlations_df.empty:
+            return []
+        return self.correlations_df.query("score < @self.threshold")["property"].tolist()
+
+    def _get_filtered_correlated_properties(self, correlated_properties: List) -> pd.DataFrame:
+        filtered_correlations_df = self.correlations_df.query("property in @correlated_properties")
+        filtered_correlations_df["property"].apply(lambda x: x.replace("_score", ""))
+        return filtered_correlations_df
+
```
Comment (elisno) [inline cleanlab/datalab/internal/adapter/imagelab.py:223]: These lines need test coverage, as this is all new code.
When will it run and when shouldn't it run?

**Review by elisno — CHANGES_REQUESTED** [review]
Please improve the test coverage of this PR.

**Review by elisno — APPROVED** [review]
LGTM!

**Pull request merged**